packages/compartment-mapper/package.json

```
@@ -21,10 +21,8 @@
   },
   "type": "module",
   "main": "./index.js",
-  "types": "./types.d.ts",
   "exports": {
     ".": {
-      "types": "./types.d.ts",
       "default": "./index.js"
     },
```

Technically this can be simplified, no?

```suggestion
    ".": "./index.js",
```